### PR TITLE
refactored assets card with new component

### PR DESCRIPTION
**Pull request by napalmpapalam**

### CODE REVIEW

File: src/scss/variables.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@ $content-padding-bottom: 2.5rem;
 $content-padding-left: 5rem;
 $content-padding-right: 6.5rem;
 $content-side-paddings-sm: 2rem;
+$list-item-margin: 2rem;
```
Comment (quinn71) [inline src/scss/variables.scss:19]: you don't use it outside the "list" component. place it there

Comment (quinn71) [reply src/scss/variables.scss:19]: also it's a gap, not margin

File: src/vue/common/Card.vue [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  content: 'content',
+  actions: 'actions',
+}
+export default {
+  name: 'card',
+  data: _ => ({
+    SLOT_NAMES,
+  }),
+  methods: {
+    isSlotDisplayed (slotName) {
```
Comment (quinn71) [inline src/vue/common/Card.vue:61]: `hasSlot`

File: src/vue/modules/assets/my-assets-explorer/index.vue [hunk 3]
```diff
@@ -1,131 +1,62 @@
...
-        <template v-if="isAssetFormShown">
-          <template slot="heading">
-            {{ 'assets.update-drawer-title' | globalize }}
-          </template>
-
-          <asset-form
-            :former="former"
-            @submitted="onAssetUpdate"
-          />
+      <template v-if="isLoadFailed">
```
Comment (quinn71) [inline src/vue/modules/assets/my-assets-explorer/index.vue:4]: looking on `load()` method, seems that it's impossible to have both `isLoaded` and `isLoadFailed` set to true

File: src/vue/common/CardList.vue [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  },
+  destroyed () {
+    window.removeEventListener('resize', this.setCardWidth)
+  },
+  methods: {
+    setCardWidth () {
+      const numberOfCards = this.getNumberOfCards()
+
+      this.styleObject = {
+        'grid-template-columns': `repeat(${numberOfCards}, minmax(25rem, 1fr))`,
```
Comment (quinn71) [inline src/vue/common/CardList.vue:59]: `repeat(autofill, minmax(25rem, 1fr)`

File: src/scss/variables.scss (hunk 1 above)
Comment (napalmpapalam) [reply src/scss/variables.scss:19]: removed this variable, and assigned value to grid-gap basic way

File: src/vue/common/Card.vue (hunk 2 above)
Comment (napalmpapalam) [reply src/vue/common/Card.vue:61]: renamed this method

File: src/vue/common/CardList.vue (hunk 4 above)
Comment (napalmpapalam) [reply src/vue/common/CardList.vue:59]: fixed

**Pull request merged**